apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -296,6 +296,17 @@ const ServiceCarouselSectionCodec = t.strict({
   cards: t.array(ServiceCardCodec),
 });
 
+const PressReleaseSectionCodec = t.strict({
+  __component: t.literal('sections.press-release'),
+  eyelet: t.union([t.string, t.null]),
+  title: t.string,
+  subtitle: t.union([t.string, t.null]),
+  body: t.string,
+  ctaText: t.union([t.string, t.null]),
+  ctaHref: t.union([t.string, t.null]),
```

Comment:
```suggestion
  link: t.union([LinkCodec, t.null]),
```
This is a link, use LinkCodec